tools/DynamoPackagesAnalyzer/Readme.txt

```diff
@@ -0,0 +1,60 @@
+This tool requires analyzebinaries experimental feature of upgrade-assistant enabled
+https://github.com/dotnet/upgrade-assistant#experimental-features
+
+This tool has three operation modes:
+
+1. zipfile
+This mode expects and arrays of packages zip archives, to be analyzed one by one
+
+Flags explanation:
+--files: used to receive the files to process
+-l: whether to lookup for package details at dynamopackages.com
+
+Example:
+DynamoPackagesAnalyzer.exe zipfile --files "some\path\package1.zip" "some\path\package2.zip" -l
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////
+
+2. directory
+This mode expects a path where the package zip archives are located, to be analyzed one by one
+
+Flags Explanation:
+-l: whether to lookup for package details at dynamopackages.com
+
+Example:
+DynamoPackagesAnalyzer.exe directory "some\path\" -l
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////
+
+3. dynamopackages
+This mode downloads and analyzes all the packages at dynamopackages.com
+
+Flags Explanation:
+No flags required
+
+Example:
+DynamoPackagesAnalyzer.exe dynamopackages
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////
+
+For all modes:
+The results files can be found at %temp%/DynamoDS or at the workspace path defined in appsettings.json
```

Comment:
so a thought... is there any reason to combine both a configuration file AND command line options to set the options? What about just using the command line options and adding some sensible defualts?

Reply:
the first version of this tool was meant to analyze only the packages at dynamopackages.com with only the appsetting.json, then we expanded the posibilities and added new operation modes, but left the appsettings.json as is, perhaps we can add new flags to override some of the configuration properties and then we could get rid of the settings file, it depends on the way we are going to use this tool